feat(BCD-419): doc for new deployment mode in bcd cli (#147)
bonitasoft/bonita-continuous-delivery-doc: md/release_notes.md

  ```

## What's new in 3.3.0 (2020-03-01)

> ### Breaking changes

Reviewer: Move the "breaking changes" part as presented on this page: https://documentation.bonitasoft.com/bcd/2.0/release_notes

The "breaking changes" part could only contain the fact that the "--development-mode" must be used to maintain the behaviour from previous versions.

Then the "What's new" could go deeper with the deployment mode table. :+1:

::: warning
The default deployment policies are changing from the previous BCD versions. 
The new deployment policies are more conservative:

|                          |3.2.1 and previous versions                          |3.3.0 and later versions                                                 |
|--------------------------|-----------------------------------------------------|-------------------------------------------------------|
|Default deployment mode   |Development                                          |Production                                             |
|Applications              |<code class="descclassname">REPLACE_DUPLICATES</code>|<code class="descclassname">REPLACE_DUPLICATES</code>  |
|Organization              |<code class="descclassname">MERGE_DUPLICATES</code>  |<code class="descclassname">IGNORE_DUPLICATES</code>   |
|Processes                 |<code class="descclassname">REPLACE_DUPLICATES</code>|<code class="descclassname">IGNORE_DUPLICATES</code>   |
|Profiles                  |<code class="descclassname">REPLACE_DUPLICATES</code>|<code class="descclassname">IGNORE_IF_ANY_EXISTS</code>|

> More information on how to upgrade you BCD [here](upgrade_bcd.md)

Reviewer: We should remove this sentence as I think we should remove the "upgrade" part. It's a new option, this does not involve file changes.
```suggestion

```

  ```

## What's new in 3.3.0 (2020-03-01)

### New features

> * Starting with release 3.3.0, BCD offer two deployment mode `development` and `production` and allow for improved policy management.

Reviewer 2: ```suggestion
* Starting with the release 3.3.0, BCD introduces deployments modes (`development` and `production`) to manage easily artifacts policies.
::: warning
The default deployment policies are changing from the previous BCD versions. More information [here](upgrade_bcd.md)
:::
```

Author: Ok for me. Any way to add your changes automatically ?

  ```

## What's new in 3.3.0 (2020-03-01)

### New features

> * Starting with the release 3.3.0, BCD introduces deployments modes (`development` and `production`) to manage easily artifacts policies.

Reviewer: ```suggestion
* With release 3.3.0, BCD introduces deployment modes (_Development_ and _Production_) to manage artifacts deployment policies more easily.
```

> * Starting with the release 3.3.0, BCD introduces deployments modes (`development` and `production`) to manage easily artifacts policies.

Reviewer: I think this would also need further details regarding this breaking change (we could mention it as a *breaking change* too btw).

Reviewer 2: What do you think of this new version?
